documentation/src/main/asciidoc/stories/assembly_marshalling_custom.adoc

```diff
@@ -0,0 +1,14 @@
+[id='alternative-marshallers']
+:context: marshaller-implementations
+= Using alternative and custom marshaller implementations
+{brandname} recommends you use ProtoStream marshalling for best performance.
```

Comment:
I'd rephrase this as "recommends you use Protobuf based marshalling with the ProtoStream marshaller for best results."